Platform/OpenCanopy/Views/BootPicker.c

```diff
@@ -1396,6 +1396,16 @@ BootPickerEntriesSet (
       case OC_BOOT_RESET_NVRAM:
         Status = CopyLabel (&VolumeEntry->Label, &GuiContext->Labels[LABEL_RESET_NVRAM]);
         break;
+      case OC_BOOT_TOGGLE_SIP:
+        if (StrCmp (Entry->Name, OC_MENU_SIP_IS_DISABLED)) {
+          Status = CopyLabel (&VolumeEntry->Label, &GuiContext->Labels[LABEL_SIP_IS_ENABLED]);
+        } else if (StrCmp (Entry->Name, OC_MENU_SIP_IS_ENABLED)) {
+          Status = CopyLabel (&VolumeEntry->Label, &GuiContext->Labels[LABEL_SIP_IS_DISABLED]);
+        } else {
+          DEBUG ((DEBUG_ERROR, "OCUI: Unsupported toggle sip name\n"));
```

Comment:
I think it should be an ASSERT, as this value should be coming from a trusted source.

Reply:
Yes - this felt wrong - thanks